docs/tables/crowdstrike_detection.md

```diff
@@ -0,0 +1,93 @@
+# Table: crowdstrike_detection
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  detection_id,
+  created_timestamp
+from
+  crowdstrike_detection
+```
+
+### List detections from the last 3 months
+
+```sql
+select
+  detection_id,
+  created_timestamp
```

Comment:
Similar to the above comment about adding more columns, once I have the detection IDs for those created in the last 3 months, what else should I know about those detections?